### PR TITLE
Signs duplicate

### PR DESCRIPTION
**Pull request by yCobanoglu**

### CODE REVIEW

**Comment by yCobanoglu** [comment]
@jlaasonen this is the newest one that got merged with master

File: src/signs/ui/search/MesZL.test.tsx [hunk 1]
```diff
@@ -27,21 +27,21 @@ bar = *zâzu*, verteilen (cf Nougayrol, RA 62 46ff.).
...
     await screen.findByText(
-      'Mesopotamisches Zeichenlexikon, Zweite revidierte und aktualisiert Auflage'
+      'Mesopotamisches Zeichenlexikon. Zweite, revidierte und aktualisierte Auflage.'
     )
   })
   it('MesZl Content', async () => {
     const { container } = render(<MesZlContent mesZl={mesZl} />)
     await screen.findByText(
-      'Mesopotamisches Zeichenlexikon, Zweite revidierte und aktualisiert Auflage'
+      'Mesopotamisches Zeichenlexikon. Zweite, revidierte und aktualisierte Auflage.'
```
Comment (jlaasonen) [inline src/signs/ui/search/MesZL.test.tsx:44]: You could extract a constant for the text so it is easier to keep updated.

File: src/signs/ui/search/MesZL.tsx [hunk 2]
```diff
@@ -1,49 +1,44 @@
...
-  mesZlNumber: string | undefined
+  mesZlRecords: SignListRecord[]
 }): JSX.Element | null {
-  const [show, setShow] = useState(false)
-  const target = useRef(null)
-
-  const formattedNumber = mesZlNumber ? ` ${mesZlNumber}` : ''
+  const renderPopover = (props) => (
+    <Popover
+      id={_.uniqueId('Citation-')}
```
Comment (jlaasonen) [inline src/signs/ui/search/MesZL.tsx:17]: Why is the id "Citation", did you just copy-paste this?

File: src/signs/ui/search/SignsSearch.tsx [hunk 3]
```diff
@@ -27,18 +27,31 @@ function sortSigns(signs: Sign[]): Sign[] {
     )
   })
 }
-
+function columnSizes(
```
Comment (jlaasonen) [inline src/signs/ui/search/SignsSearch.tsx:30]: What is the point of the column sizes? Wouldn't it be more straightforward to use a table?
It is also weird to interleave rows and list items.

**Comment by jlaasonen** [comment]
When merging, remember to update the commit message. "Signs duplicate (will be merged) the original will be left for review" is not very informative.

**Pull request merged**